bin/wait_then_run

```diff
@@ -24,6 +24,7 @@ until curl --silent ipfs:5001; do
 done
 
 mkdir -p minio/escrow-results
+mkdir -p minio/escrow-public-results
```

Comment:
any reason to have an additional bucket here? I think, instead of adding additional logic and `if's` it is better to have 1 bucket and a bunch of configuration that controls the behavior: like "enable encryption", "show the full URL" and etc

the code will be much more simpler and straightforward, what do you think?

Reply:
Hi @vkomodey , 

the request was that when encryption is off, we'll show the full qualified URL - [IMI-2282](https://intuitionmachines.atlassian.net/browse/IMI-2282) - and final results persisted in blockchain plainly and the full URL publicly exposed to user to download the files whenever he wants.

However, to support if a file can be publicly exposed in AWS S3, you have to configure the bucket to support that, which makes the bucket vulnerable with possibility to expose non-public chunk data. So, by determining that the information is public and chunk file is downloadable, it is reasonable (and secure) to send data to bucket that supports public exposure of its files stored there.

So, I believe we should move the discussion to the task to determine the best approach. And, I completely agree, using one single bucket would make everything simpler and code less complex.

Reply:
@vkomodey from my perspective, there's this little extra complexity in configuration. But it's essentially solved by copy-pasting the s3 creds if you'd like the same bucket. 

On the other hand, if you would like a better security at some point, by having another bucket, you'd be out of luck, and need dev time. 

But the decision, I think, can be yours here.

Reply:
Agree. Let's have several buckets